src/Storage/Storage.Management.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -965,3 +965,54 @@ function Test-StorageAccountManagementPolicy
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test Test-NewSetAzureStorageAccount_LargeFileShare
+.DESCRIPTION
+SmokeTest
+#>
+function Test-NewSetAzureStorageAccount_LargeFileShare
```

Comment:
Why using `_` here instead of using CamelCase？

Reply:
same as above.